Documentation/CRDs/Shared-Filesystem/ceph-filesystem-crd.md

```diff
@@ -135,13 +135,13 @@ spec:
     # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
     peers:
       secretNames:
-        - secondary-cluster-peer
+    # Add the secret name if it already exists else specify the empty list here.
     # specify the schedule(s) on which snapshots should be taken
     # see the official syntax here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-schedules
     snapshotSchedules:
       - path: /
         interval: 24h # daily snapshots
-        startTime: 11:55
+        startTime: YYYY-MM-DDT11:55:00
```

Comment:
WHats this format of date,

IG you probably mean formats like this `YYYY-MM-DDT` and example as `11:55:00`,
If yes, kindly separate it out

Reply:
Is `YYYY-MM-DD` a valid date, or will this also cause the example to fail? Do we need to put a date in like `2020-01-01T11:55:00`?

Reply:
Hi @BlaineEXE nope i don't think `YYYY-MM-DD` is a valid date format because after trying `2022-07-12` as well i got the following error from the operator logs : 
```
2022-07-13 08:58:00.814090 E | ceph-file-controller: failed to reconcile failed to configure mirroring for filesystem "myfs".: failed to add snapshot schedules on filesystem "myfs": failed to add snapshot schedule every "24h" to ceph filesystem "myfs" on path "/". . Error ENOENT: The date string 2022-07-12 does not match the required format
            %Y-%m-%dT%H:%M:%S. For more flexibel date parsing upgrade to
            python3.7 or install
            https://github.com/movermeyer/backports.datetime_fromisoformat: exit status 2
2022-07-13 08:58:00.814293 E | ceph-file-controller: failed to reconcile CephFilesystem "rook-ceph/myfs". failed to configure mirroring for filesystem "myfs".: failed to add snapshot schedules on filesystem "myfs": failed to add snapshot schedule every "24h" to ceph filesystem "myfs" on path "/". . Error ENOENT: The date string 2022-07-12 does not match the required format
            %!Y(MISSING)-%!m(MISSING)-%!d(MISSING)T%!H(MISSING):%!M(MISSING):%!S(MISSING). For more flexibel date parsing upgrade to
            python3.7 or install
            https://github.com/movermeyer/backports.datetime_fromisoformat: exit status 2

```

Reply:
We need to put the date in the format like `2022-07-13T11:55:00`